### PR TITLE
NCDA for CHW - questions changes at tasks

### PR DESCRIPTION
**Pull request by anvmn**
#788 
#783 
#785 
#786 
#784

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-07-17 10-56

**Comment by anvmn** [comment]
Peek 2023-07-17 10-59

**Comment by anvmn** [comment]
Peek 2023-07-17 23-12

**Comment by anvmn** [comment]
Peek 2023-07-18 10-55

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -9543,6 +9594,19 @@ translationSet trans =
                     , kinyarwanda = Nothing
                     }
 
+                MealsAtRecommendedTimes ->
+                    { english = " Does the child eat at the recommended times per day"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]:
```suggestion
                    { english = "Does the child eat at the recommended times per day"
```

**Review by mariano-dagostino — APPROVED** [review]

**Pull request merged**